app/assets/stylesheets/config/_import.scss

```diff
@@ -25,17 +25,9 @@ $colors: (
   'base-0': var(--base-0),
   'base-inverted': var(--base-inverted),
   'accent-brand': var(--accent-brand),
-  'accent-brand-darker': var(--accent-brand-darker),
-  'accent-brand-lighter': var(--accent-brand-lighter),
```

Comment:
What does removing these color variables impact? Are they being removed because they're not really being used?